docs/api-reference.mdx

```diff
@@ -327,6 +328,65 @@ We support a few different formats for user data, for example the default format
 </details>
 :::
 
+### Update 1-Click Credentials
+
+> This endpoint is to be used for updating the 1-click credentials passed to your brand in the 1-click non-hosted and 1-click trusted flows.
+
+**Method**: `POST` <br />
+**Path**: `/1-click/{uuid}/credentials`
+
+You need to provide:
+
+1. **The 1-click uuid**
+2. **The credential values in same format they were provided.** See example below.
+
+<SimplifiedCredentialsExample />
+
+This is an optional endpoint only relevant for the 1-click [non-hosted](/#non-hosted-variant) and [trusted](/#trusted-variants) variants. As the user data is sent directly to your application without interaction with Verified Inc's Web Wallet—part of the hosted solution—the user never gets a chance to update any pre-populated data provided. This editing and enrichment can only occur within a UI/UX that you manage and implement. This endpoint can be used to pass that information back to Verified Inc.
```

Comment:
```suggestion
This is an optional endpoint only relevant for the 1-click [non-hosted](/#non-hosted-variant) and [trusted](/#trusted-variants) variants. Because we send the user's data directly to your application, without the user using our web Wallet (which is part of the hosted variant), the user never gets a chance to update any of the data that's autofilled for them. If you allow them to editing and/or enrich their data on your side,  within a UI/UX that you manage and implement, you can use this endpoint to pass that information back to us (and earn revenue share as a result).
```